onelogin/get.go

```diff
@@ -107,30 +109,80 @@ func Get(app, provider string) (*awsprovider.Credentials, error) {
 		fmt.Scanln(&selection)
 
 		deviceID = fmt.Sprintf("%v", devices[selection-1].DeviceId)
+		deviceType = devices[selection-1].DeviceType
+
 	} else {
 		deviceID = fmt.Sprintf("%v", devices[0].DeviceId)
+		deviceType = devices[0].DeviceType
 	}
 
-	fmt.Print("Please enter the OTP from your MFA device: ")
-	var otp string
-	fmt.Scanln(&otp)
+	var samlAssertion string
+
+	var pushSupported = false
+
+	if deviceType == "OneLogin Protect" {
+		// push is supported by the selected OTP device
+		// if other are supporting push they should be added here.
```

Comment:
We'll I wasn't sure about OneLogin SMS. As it's not enabled in our corp, I wouldn't know.

Reply:
Sounds from the docs that only Protect currently supports push. SMS has a different mechanism and since we haven't implemented it in the app yet, I guess we can simply ignore this case.
In any case, I think this comment should be edited.